Library/Homebrew/service.rb

```diff
@@ -267,5 +277,28 @@ def to_systemd_unit
 
       unit + options.join("\n")
     end
+
+    # Returns a `String` systemd unit timer.
+    # @return [String]
+    sig { returns(String) }
+    def to_systemd_timer
+      timer = <<~EOS
+        [Unit]
+        Description=Homebrew generated timer for #{@formula.name}
+
+        [Install]
+        WantedBy=timers.target
+
+        [Timer]
+        Persistent=true
```

Comment:
https://www.freedesktop.org/software/systemd/man/systemd.timer.html#Persistent=

> Note that this setting only has an effect on timers configured with OnCalendar=.

Should we use `OnCalendar` here instead? I guess that would be useful for us when the service for is cron.

Reply:
That's going to be my next PR. I'll make sure that this only applies on cron then